partcad/src/partcad/utils_ocp.py

```
@@ -0,0 +1,106 @@
+
+    vertices: List[Tuple[float, float, float]] = []
+    triangles: List[Tuple[int, int, int]] = []
+    offset = 0
+
+    for f in get_faces(shape):
+        loc = TopLoc_Location()
+        poly = BRep_Tool.Triangulation_s(f, loc)
+        if poly is None:
+            continue
+        Trsf = loc.Transformation()
+        reverse = True if f.Orientation() == TopAbs_Orientation.TopAbs_REVERSED else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)